### PR TITLE
(appengine) change model for upsert load balancer description

### PR DESCRIPTION
**Pull request by danielpeach**
@lwander or @duftler please review.

### CODE REVIEW

File: clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/UpsertAppengineLoadBalancerAtomicOperationConverter.groovy [hunk 1]
```diff
@@ -32,6 +33,13 @@ class UpsertAppengineLoadBalancerAtomicOperationConverter extends AbstractAtomic
   }
 
   UpsertAppengineLoadBalancerDescription convertDescription(Map input) {
-    AppengineAtomicOperationConverterHelper.convertDescription(input, this, UpsertAppengineLoadBalancerDescription)
+    UpsertAppengineLoadBalancerDescription description = AppengineAtomicOperationConverterHelper
+      .convertDescription(input, this, UpsertAppengineLoadBalancerDescription) as UpsertAppengineLoadBalancerDescription
+
+    if (description.splitDescription) {
+      description.split = AppengineModelUtil.convertTrafficSplitDescriptionToTrafficSplit(description.splitDescription)
```
Comment (lwander) [inline clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/UpsertAppengineLoadBalancerAtomicOperationConverter.groovy]: Just wondering, why is this needed? Why not supply clouddriver with the right format?

Comment (danielpeach) [inline clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/converters/UpsertAppengineLoadBalancerAtomicOperationConverter.groovy]: We talked about this offline, and I have mixed feelings about it. It's ugly to make this transformation here, but since enable, disable, upsertLoadBalancer, and the App Engine API all represent a load balancer's allocations as a map, I think this solution limits the total number of transformations.
